src/scenes/AcceleratorRouter/Cohorts/CohortCellRenderer.tsx

```diff
@@ -19,5 +20,9 @@ export default function CohortCellRenderer(props: RendererProps<TableRowType>):
     );
   }
 
+  if (column.key === 'participantIds') {
+    return <CellRenderer {...props} value={<TextTruncated stringList={(value as string[]) || []} />} />;
+  }
+
```

Comment:
I'm sure we want to show participant names here rather than participant ids, just did this as a placeholder start for now.

Reply:
Turns out in the designs we're showing a count rather than a list of names:

<img width="749" alt="Screenshot 2024-03-26 at 11 24 06 AM" src="https://github.com/terraware/terraware-web/assets/1474361/751dc4da-6593-4d84-8935-50d6fc5be3c8">